draft-ietf-httpbis-digest-headers.md

```diff
@@ -962,6 +1055,29 @@ Specification document(s):  {{digest-header}} of this document
    and adds two more algorithms: ID-SHA-256 and ID-SHA-512 which allows to
    send a checksum of a resource representation with no content codings applied.
 
+8. How can I test digest output? Use the following python3 function.
```

Comment:
Editorial suggestion:

It would be nice to split the line after the question, and provide a little bit more decription in the. For example

```
8. How can I generate and validate the Digest values shown in the examples throughout this document?

The following python3 script can be used to generate digests using SHA algorithms for a range of encodings. Note that these are formatted as base64. This function could be adapted to other algorithms and should take into account their specific formatting rules.

Reply:
addressing in #938